hazelcast/src/main/java/com/hazelcast/jet/impl/SubmitJobTargetMemberFinder.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.impl;
+
+import com.hazelcast.client.impl.clientside.HazelcastClientInstanceImpl;
+import com.hazelcast.cluster.Member;
+import com.hazelcast.jet.JetException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.UUID;
+
+public class SubmitJobTargetMemberFinder {
+
+    /**
+     * Pick a random cluster member
+     */
+    UUID getRandomMemberId(HazelcastClientInstanceImpl client) {
+        ArrayList<Member> memberList = new ArrayList<>(client.getClientClusterService().getMemberList());
+        return getRandomIdFromList(memberList);
+    }
+
+    UUID getRandomIdFromList(ArrayList<Member> memberList) {
+        if (memberList.isEmpty()) {
+            throw new JetException("Cluster member list is empty");
+        }
+        Collections.shuffle(memberList);
```

Comment:
`memberList.get(random.nextInt(memberList.size()))`?

Reply:
The code was like that before but Sonar is failing the usage of Random class. So I have changed it like this